ocropus-rtrain

```diff
@@ -156,7 +156,8 @@ def save_lstm(fname,network):
         network.lstm.save(fname)
     else:
         if args.strip:
-            network.clear_log()
+            print yellow('saving stripped network (without temporary variables)...')
```

Comment:
The function `yellow` is defined here https://github.com/tmbdev/ocropy/blob/80f05404b5f0cbdbcfa60550dafcd8749685be65/ocrolib/common.py#L826 . Do we need that?